README.md

```diff
@@ -1,2 +1,47 @@
-# ruby-sinatra-sample-provider
-A minimal provider using Sinatra
+# Sample Manifold Provider (Ruby/Sinatra)
+
+This repo contains a a minimal provider using Sinatra and Ruby.
+It provides digital cat bonnets as a service; fun!
+
+# Testing the app with Grafton
+
+Grafton is the test framework used to verify your provider implementation is
+correct.
+
+Contact [support@manifold.co](mailto:support@manifold.co) to get access to
+Grafton.
+
+
+To use Grafton to verify this sample provider:
+
+```bash
+# create a test master key for grafton to use when acting as Manifold
+# this file is written as masterkey.json
+grafton generate
+
+# Set environment variables to configure the test app
+# VERIFIER_KEY is the public_key portion of masterkey.json
+export VERIFIER_KEY="TtziSVE/9lnZ7fRYhWtGZpuXUKJ82FunK9rM0IkuP/0"
```

Comment:
grafton generate (or grafton init, when/if we change it) should print out the values you'll need to use.